4.0/asvs.py

```diff
@@ -203,3 +215,18 @@ def to_csv(self):
         writer.writerows(self.asvs_flat)
 
         return si.getvalue()
+
+    def to_opencontrol(self):
+        ''' Returns opencontrol yaml '''
+
+        opencontrol_result=yaml.dump({'name':f"{self.asvs['ShortName']}-{self.asvs['Version']}"})
+        for key in self.sorted_nicely(self.asvs_opencontrol.keys()):
+            opencontrol_result+=yaml.dump({
+                key:{
+                    "family":self.asvs_opencontrol[key]["family"],
```

Comment:
The output matches the standard schema: https://github.com/opencontrol/schemas/blob/master/kwalify/standard/v1.0.0.yaml

It would also be possible to generate the certification.yamls for each of the L1/L2/L3 certifications.